assets/src/blocks/Accordion/AccordionEditor.js

```diff
@@ -0,0 +1,314 @@
+                        keepPlaceholderOnFocus={true}
+                        withoutInteractiveFormatting
+                        // characterLimit={100}
+                        multiline="false"
+                      />
+                    </div>
+                    <div className="panel">
+                      <RichText
+                        // id={attributes.accordion_id, index+1}
+                        tagName="p"
+                        className="accordion-text"
+                        placeholder={__('Enter description here', 'planet4-blocks-backend')}
+                        value={attributes.accordion_text}
+                        onChange={this.toAttribute('accordion_text')}
+                        keepPlaceholderOnFocus={true}
+                      />
+                    <Tooltip text={__('Add button text or leave empty to hide it on the front page.', 'planet4-blocks-backend')}>
+                      <div className="btn btn-secondary btn-block">
+                        <RichText
+                          tagName="div"
+                          placeholder={__('Add button text or leave empty to hide it.', 'planet4-blocks-backend')}
+                          value={attributes.accordion_btn_text}
+                          // href={attributes.accordion_btn_url}
+                          onChange={ this.toAttribute('accordion_btn_text') }
+                          keepPlaceholderOnFocus={true}
+                          withoutInteractiveFormatting
+                          multiline="false"
+                        />
+                      </div>
+                    </Tooltip>
+                    </div>
+                  </div>
+
+                </div>
+              );
+            })}
+            {/* <div className="container">
+              { accordion_rows.length < 5 && (
+                <Tooltip text={__('Add Row', 'planet4-blocks-backend')}>
+                  <button
+                    className={
+                      "btn btn-outline-secondary btn-sm btn-mnml"
+                    }
+                    onClick={this.props.addAccRow}
+                  >
+                    {__('Add Row', 'planet4-blocks-backend')} <Dashicon icon={"plus"} size={12} />
+                  </button>
+                </Tooltip>
+              )}
+
+              { accordion_rows.length > 0 && (
+                <Tooltip text={__('Remove Row', 'planet4-blocks-backend')}>
+                  <button
+                    className={ "btn btn-outline-secondary btn-sm btn-mnml" }
+                    onClick={this.props.removeAccRow}
+                  >
+                    {__('Remove Row', 'planet4-blocks-backend')} <Dashicon icon={"minus"} size={12} />
+                  </button>
+                </Tooltip>
+              )}
+            </div> */}
+          </div>
+        </section>
+
+        <AccordionFrontend isEditing {...attributes} />
```

Comment:
From a quick look at `AccordionFrontend` it looks like pretty much nothing is rendered if `isEditing` is true, so we probably don't need to add it here...? 🤔